_src/static/sass/utilities/_mixins.scss

```diff
@@ -0,0 +1,275 @@
+// ----------------------------
+// #MIXINS
+// Font Sizing
+// Line Height
+// Placeholder Styling
+// Support (Flex)
+// Headings (Mixin)
+//
+// #EXTENDS
+// Headings (Extend)
+// Polyfill (Flex)
+// Windows
+// Visual Disabled
+// Layering/Zindexing
+// Clear Children
+// Lists
+// Icon Fonts
+// Accessibility
+// Text Handling
+// Content Handling
+// ----------------------------
+
+
+// #MIXINS
+// ----------------------------
+// Font Sizing
+// ----------------------------
+@mixin font-size($size: 16) {
+    font-size: ($size/10) + rem;
+}
+
+
+// ----------------------------
+// Line Height
+// ----------------------------
+@mixin line-height($size: 1.5rem) {
+    line-height: (($size/10) * 1.48) + rem;
+}
+
+
+// ----------------------------
+// Placeholder Styling
+// ----------------------------
+@mixin placeholder($color) {
```

Comment:
This is not needed as its covered by Bourbon's placeholder utility - http://bourbon.io/docs/#placeholder. Please remove.